_fixtures/helloworld.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
We already have other fixtures that don't do a whole lot, in the interest of avoiding fixture proliferation I suggest using increment.go or testargs.go